### DISCUSSION TITLE
LDAP Issues

### DISCUSSION THREAD

**Discussion by KeptnArgo** [kimai/kimai#2502] [Questions & Answers]
**Describe the bug**
From what i've got the new LDAP config is the same as for 1.7 but the framework is just different. I've followed the guide from https://www.kimai.org/documentation/ldap.html, copied the local.yaml from my older 1.7 installation and did the whole re-build cache thing. but for some reasons there aren't any requests to the ldap server, not even dns requests for it. Also there are no logs related to ldap neither in prod nor in dev 

**To Reproduce**
1. Follow the LDAP guide.

**Logfile**
```
Add the last lines from your logfile at "var/log/prod.log" or "Doctor > Logs", around the time when the problem happened.
```

**Additional context**
 - Kimai version: 1.13
 - PHP version: 7.2.24

**Comment by kevinpapst** [comment]
LDAP is working as expected. Its used by many many customers. You have a configuration issue.

**Reply by KeptnArgo** [reply]
so have there been any configuration changes from 1.7 to 1.13 i've overlooked ? i've copied the local.yaml from a working 1.7 instance (with ldap login using the zend) to a new 1.13 instance and followed https://www.kimai.org/documentation/ldap.html to install laminas-ldap.
